content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md

```diff
@@ -156,8 +173,12 @@ kubectl get all
 ---
 
 ### To monitor the performance metrics on prometheus
-Follow the below site:
-
+Open the `configMap.yaml` file 
+```bash
+# Uncomment the line in configMap.yaml file
+performance_exporters = PerformanceExporterPrometheus, PerformanceExporterKafka
```

Comment:
Are both exporters needed or only the `PerformanceExporterPrometheus` is needed?
During SODA installation with delfin we enable the exporter that is required but not both.

Reply:
changed